airflow/www/views.py

```diff
@@ -151,18 +151,23 @@ def get_date_time_num_runs_dag_runs_form_data(request, session, dag):
 #                                    Error handlers
 ######################################################################################
 
-def circles(error):
+# noinspection PyUnusedLocal
```

Comment:
Is this just for IntelliJ? Pycharm? If yes, can we remove please

Reply:
Indeed. I was too lazy. I replaced all those with 'noqa's and will keep on doing it.